content/en-US/api/fontVariantCaps.md

```diff
@@ -0,0 +1,101 @@
+---
+recipe: api-interface
+title: 'CanvasRenderingContext2D.fontVariantCaps'
+mdn_url: /en-US/docs/Web/API/CanvasRenderingContext2D/fontVariantCaps
+specifications: [[specURL]]#[[HeadingAnchor]]
+browser_compatibility: api.CanvasRenderingContext2D.fontVariantCaps
+---
+
+**When this feature ships, the content below will live on MDN under
+[developer.mozilla.org/en-US/docs/Web/CanvasRenderingContext2D/fontVariantCaps](https://developer.mozilla.org/en-US/docs/Web/CanvasRenderingContext2D/fontVariantCaps).**
+
+## Description
+
+The `fontVariantCaps` property of `CanvasRenderingContext2D` allows developers
+to control the use of alternate glyphs for capital letters. If multiple sizes 
+of capital letter glyphs are available for the chosen font, this 
+`CanvasRenderingContext2D` property will choose the one with appropriate size. 
+Otherwise it synthesizes small-caps by using uppercase glyphs. The default 
+value is `normal`.
+
+
+## Syntax
+
+`CanvasRenderingContext2D.fontVariantCaps = "normal" || "small-caps" || "all-small-caps" || "petite-caps" || "all-petite-caps" || "unicase" || "titling-caps"`
+`var fontVariantCaps = CanvasRenderingContext2D.textRendering`
+
+### Values
+
+A `string` representing current `fontVariantCaps` value. The possible values are:
+
+#### Options
+
+`"normal"`
+Deactivates of the use of alternate glyphs.
+
+`"small-caps"`
+Enables display of small capitals (OpenType feature: `smcp`). Small-caps glyphs
```

Comment:
Please be consistent about whether you use 'small caps' or 'small capitals'.

Reply:
I should have been clearer. 

- When referring to the property specifically, use `small-caps`. 
- When referring to the concept use "small caps".

The later is based on the fact that "small caps" is what's used on MDN.